pr_check.sh

```diff
@@ -23,11 +23,15 @@ mkdir -p "$DOCKER_CONF"
 docker login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
 
 RESPONSE=$( \
-        curl -Ls -I -o /dev/null -w "%{http_code}" -H "Authorization: Bearer $QUAY_API_TOKEN" \
-        https://quay.io/api/v1/repository/cloudservices/clowder-base/tag/$BASE_TAG/images \
+        curl -Ls -H "Authorization: Bearer $QUAY_API_TOKEN" \
+        "https://quay.io/api/v1/repository/cloudservices/clowder-base/tag/?specificTag=$BASE_TAG" \
     )
-    echo "received HTTP response: $RESPONSE"
-if [[ $RESPONSE != 200 ]]; then
+
+echo "received HTTP response: $RESPONSE"
+
+TAGS_LENGTH=$(echo $RESPONSE | jq '.tags | length')
```

Comment:
To filter out expired tags:

```suggestion
TAGS_LENGTH=$(echo $RESPONSE | jq '[ .tags[] | select(.end_ts == null) ] | length')
```